Doc/library/email.utils.rst

```diff
@@ -26,6 +26,8 @@ module:
 
     .. versionadded:: 3.3
 
+    .. deprecated:: 3.11
+       Use of the ``isdst`` argument is deprecated.
```

Comment:
I just noticed the style for parameters is italics (for example see the docstring):
```suggestion
       Use of the *isdst* argument is deprecated.
```